snapcraft.yaml

```diff
@@ -27,9 +29,18 @@ parts:
       - lib
       - sbin
     stage-packages:
+      - coreutils
+      - dosfstools
       - e2fsprogs
       - fakeroot
       - mtools
       - python3-debian
       - python3-parted
       - util-linux
+    install: |
+      TRIPLET="$(uname -p)-linux-gnu"
+      mv $SNAPCRAFT_PART_INSTALL/usr/lib/$TRIPLET/* $SNAPCRAFT_PART_INSTALL/usr/lib/
```

Comment:
I didn't know cross-building of snaps is possible and we can safely assume host arch = snap arch. If that's not the case, I'll just use the same trick I did for python then.